src/vcpkg/install.cpp

```diff
@@ -795,6 +799,12 @@ namespace vcpkg::Install
                                  "",
                                  "Error: The option --{option} is not supported in manifest mode.");
 
+    DECLARE_AND_REGISTER_MESSAGE(StampNotChanged,
+                                 (msg::option),
+                                 "",
+                                 "Installation skipped. Everything seems to be installed. To disable this check, don't "
```

Comment:
I really do not love this message; it should be a simple "No changes detected" or something, there's no need to make it complex.